torch/csrc/jit/script/script_type_parser.h

```diff
@@ -28,6 +33,7 @@ class TORCH_API ScriptTypeParser {
   at::TypePtr subscriptToType(
       const std::string& typeName,
       const Subscript& subscript) const;
+  ResolverPtr resolver_ = nullptr;
```

Comment:
Why would this ever need to be null?

Reply:
If we ever wanted to use a "bare" ScriptTypeParser without an outer environment to look up type names in. Doesn't seem super likely, but there's no reason why we should require a resolver.

Reply:
not used yet?